simbev/car.py

```diff
@@ -876,10 +888,13 @@ def export(self, region_directory, simbev):
                 ] / (post_event_len * simbev.step_size / 60)
 
                 # fit first row event to start at time step 0
-                activity.at[activity.index[0], "event_start"] = 0
-                activity.at[activity.index[0], "event_time"] = post_event_len
+                activity.at[activity.index[0], "event_start"] = np.int32(0)
+                activity.at[activity.index[0], "event_time"] = np.int32(post_event_len)
                 activity.at[activity.index[0], "timestamp"] = simbev.start_date_output
 
+                activity["event_start"] = activity["event_start"].astype("int32")
+                activity["event_time"] = activity["event_time"].astype("int32")
```

Comment:
Aren't the last 2 lines enough? What exactly does the casting to int32 on assignment do?

Reply:
Thats right. After the last Commit the casting is not done anymore, because the datatype gets casted on assignement already.